src/generic/course-card/types.ts

```diff
@@ -27,6 +27,6 @@ export interface Course {
 }
 
 export interface CourseCardProps {
-  course?: Course;
+  original?: Course;
```

Comment:
Could you explain this name change? It's not clear to me what `original` implies here.